Formula/rustfmt.rb

```diff
@@ -0,0 +1,26 @@
+class Rustfmt < Formula
+  desc "Format Rust code"
+  homepage "https://rust-lang.github.io/rustfmt/"
+  url "https://github.com/rust-lang/rustfmt/archive/refs/tags/v1.4.38.tar.gz"
+  sha256 "4856cac540a6d894bc019285b690cebd2076aa8067bd8509ec176645c47a31bf"
+  license any_of: ["MIT", "Apache-2.0"]
+  head "https://github.com/rust-lang/rustfmt.git", branch: "master"
+
+  depends_on "rust" => [:build, :test]
+  depends_on "rustup-init" => :build
```

Comment:
```suggestion
  depends_on "rustup-init" => :build
  depends_on "rust" => :test
```
Does this work? It doesn't seem like you should need `rust` at runtime given that you're installing everything else with `rustup`.